docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -458,9 +458,9 @@ The `locale` prop of the `LocalizationProvider` component have been renamed `ada
  </LocalizationProvider
 ```
 
-## Component slots / component slots props
+## Component slots / component slot props
```

Comment:
How do you suggest/plan to handle `components` and `componentsProps` situation in this migration file?
Should we go ahead and rename it here as well? 🤔 
Or we want to do a separate migration section mentioning the general rename of these props? 🤔